dir.props

```diff
@@ -271,9 +271,8 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <RoslynVersion>3.0.0-beta4-final</RoslynVersion>
-    <RoslynPackageName>Microsoft.Net.Compilers</RoslynPackageName>
-    <RoslynPackageName Condition="'$(RunningOnCore)' == 'true'">Microsoft.NETCore.Compilers</RoslynPackageName>
+    <RoslynVersion>3.1.0-beta3-19213-02</RoslynVersion>
+    <RoslynPackageName>Microsoft.Net.Compilers.Toolset</RoslynPackageName>
```

Comment:
We now have a single meta package that supports both .NET Core and Desktop. It contains the logic for switching between the two based on which msbuild is running and makes switching between the command line and VS work